catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/ExportCommand.java

```diff
@@ -162,9 +162,7 @@ protected Object executeWithSubject() throws Exception {
         }
 
         if (delete && !force) {
-            console.println(
-                    "This action will remove all exported metacards and content from the catalog. Are you sure you wish to continue? (y/N):");
-            String input = getUserInputModifiable().toString();
+            final String input = session.readLine("This action will remove all exported metacards and content from the catalog. Are you sure you wish to continue? (y/N):", null);
```

Comment:
✏️  Nitpick: Could you go ahead and change that N to either be lowercase or the Y to be uppercase so that they are the same case?